tests/Version_test.cpp

```diff
@@ -140,10 +145,10 @@ class VersionTest : public QObject {
 
     void test_flexVerTestVector()
     {
-        QFETCH(QString, first);
-        QFETCH(QString, second);
-        QFETCH(bool, lessThan);
-        QFETCH(bool, equal);
+        QString first = *static_cast<QString*>(QTest ::qData("first", ::qMetaTypeId<typename std ::remove_cv<QString>::type>()));
```

Comment:
not a suggestion since i'm pretty sure these work...but dear god c++ scares me sometimes

Reply:
ugg, my bad, this is the expanded QFETCH Macro (I was trying to trace the use-after-free, this is how I figured out I was a pointer not a copy).